scripts/pythonscripts/sync_postgres_to_mssql.py

```diff
@@ -0,0 +1,228 @@
+import psycopg2
+import pyodbc
+import pandas as pd
+import numpy as np
+from concurrent.futures import ThreadPoolExecutor
+from typing import List
+import time
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+def get_schema_tables():
+    pg_conn = psycopg2.connect(
+        host=os.getenv('PG_HOST'),
+        database=os.getenv('PG_DATABASE'),
+        user=os.getenv('PG_USER'),
+        password=os.getenv('PG_PASSWORD'),
+        port=os.getenv('PG_PORT', '5432')
+    )
+    # Get all tables from the schema
+    cursor = pg_conn.cursor()
+    cursor.execute("""
+        SELECT table_name 
+        FROM information_schema.tables 
+        WHERE table_schema = 'curated_kawasaki_registry'
+        AND table_name NOT LIKE 'stg_%'
+        AND table_name NOT LIKE 'int_%'
+    """)
+    tables = [table[0] for table in cursor.fetchall()]
+    print(tables)
```

Comment:
In general, consider using the `logging` package instead of `print`. It is generally preferable for production ready code: https://www.geeksforgeeks.org/difference-between-logging-and-print-in-python/

```
import logging
logger = logging.getLogger(__name__)

logger.info("LOG HERE")
```